.env/.env.template

```diff
@@ -11,11 +11,9 @@ CLIENT_API_KEY=
 CORS_ENABLE_WHITELIST=false
 
 # GRAPHQL
-GRAPHQL_URL=http://localhost:9000/v1/graphql
-GRAPHQL_SECRET=myGraphQlSecret
+GRAPHQL_URL_HET_ARCHIEF=http://localhost:9000/v1/graphql
+GRAPHQL_SECRET_HET_ARCHIEF=myGraphQlSecret
```

Comment:
Why does this need to be namespaced in the implementing application? The admin core will only ever require one set of url/secret, it'll never need both `GRAPHQL_URL_HET_ARCHIEF` and `GRAPHQL_URL_AVO`.

Reply:
there are multiple graphql databases:
* org api
* logging api
* hetarchief
* avo

This suffix is to make a clear difference between the hetarchief database and the logging database.